src/cromshell/utilities/submissions_file_utils.py

```diff
@@ -0,0 +1,37 @@
+from enum import Enum
+
+
+class WorkflowDatabaseColumns(Enum):
+    """Enum holding mutable and immutable all_workflow_database.tsv column headers"""
+
+
+# Immutable class needs to be placed
+class ImmutableSubmissionFileHeader(WorkflowDatabaseColumns):
+    Date = "DATE"
+    Cromwell_Server = "CROMWELL_SERVER"
+    Run_ID = "RUN_ID"
+    WDL_Name = "WDL_NAME"
+
+
+class MutableSubmissionFileHeader(WorkflowDatabaseColumns):
+    Status = "STATUS"
+    Alias = "ALIAS"
+
+
+def get_submission_file_headers(
+    workflow_database_columns=WorkflowDatabaseColumns,
```

Comment:
What if a user wants all headers?
A possible solution is to
* here, check the input isn't empty or None, and
* have another function, e.g. `get_submission_file_all_headers`.

Reply:
But this function does return all the headers in the WorkflowDatabaseColumns Enum?

Reply:
Sorry, then I am confused, what's the purpose of having `workflow_database_columns` as input parameter?

Reply:
`workflow_database_columns` is default for the function to pull in WorkflowDatabaseColumns enum. Having it as a default parameter makes it easier later if we want to change the default during unit tests.

Reply:
Let's talk during todays meeting. I'm not fully understanding yet.